pkg/operator/deploy/deploy.go

```diff
@@ -128,6 +129,11 @@ func (o *operator) resources() ([]runtime.Object, error) {
 		return nil, err
 	}
 
+	vNetID, _, err := subnet.Split(o.oc.Properties.MasterProfile.SubnetID)
```

Comment:
maybe `vnetID`?

Reply:
fixed

Reply:
seems like I wasn't right about this one. We are all over the shop with this particular field/variable name (`git grep -i vnetid`) and Azure does capitalize N, e.g. https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-faq
Up to you how you would like to see it.

Reply:
`vnetID` is fine. Most go samples out there use `vnet`. E.g. [here](https://github.com/Azure-Samples/azure-sdk-for-go-samples/blob/master/network/network_test.go).